lib/sensu/api/process.rb

```diff
@@ -334,7 +334,17 @@ def resolve_event(event_json)
           unless clients.empty?
             clients.each_with_index do |client_name, index|
               settings.redis.get("client:#{client_name}") do |client_json|
-                response << MultiJson.load(client_json)
+                begin
+                  response << MultiJson.load(client_json)
+                rescue MultiJson::ParseError
```

Comment:
We could just have a `if client_json.nil?` conditional, instead of catching the parser errors.